### ISSUE TITLE
Ignored local/internal dependencies

### ISSUE THREAD

**Issue by rodrigokono** [goenning/dotnet-outdated#3]
So, In cases where have a dependencies local or internal in project.json, not work. 
A exception like "not found" is notified.

The suggest is to ignore not found errors and go on.
Tks

**Comment by goenning** [comment]
I agree with that for local dependencies. But when you say internal, you mean a private registry? I don't experience with that, but I guess we could query the private nuget like we do with the public.

**Comment by rodrigokono** [comment]
Yes. Are private packages.
My private packages are in myget and I guess that get the error when try to get the private packages.
